moveit2_tutorials.repos

```diff
@@ -1,8 +1,4 @@
 repositories:
-  moveit2_tutorials:
-    type: git
-    url: https://github.com/ros-planning/moveit2_tutorials.git
-    version: main
```

Comment:
Why did you remove this repo? This will break the instructions in the README. We only need to remove rosdoc_lite and genmsg from this file.

Reply:
Your right - my bad